### DISCUSSION TITLE
Is it possible to build fully static binary of program using ImGui?

### DISCUSSION THREAD

**Discussion by ximik69** [ocornut/imgui#7085] [New Users Build/Link/Run issues ONLY!]
Hi everyone.
I am looking whether ImGui fits my purpose.
I am looking for some simple gui lib for c/c++ with 10+ years of forward and backward compatibility under linux regardless of kernel, glibc, x11 and other libs versions. I see the only viable option to achieve that - static binaries. Optionally the same approach for windows.
Is it possible to build fully static binary of program using ImGui?
If yes, can I ask you for the recipe?

All the best,
Igor.

**Comment by ocornut** [comment]
Yes. Nothing special, just build every lib you use as static.

**Reply by ximik69** [reply]
Ok, thank you!
